src/graph/definitions/advertiser.graphql

```diff
@@ -65,6 +65,12 @@ input AdvertiserSortInput {
 
 input AdvertiserPayloadInput {
   name: String!
+  notify: AdvertiserContactsInput
+}
+
+input AdvertiserContactsInput {
+  internal: [String!]
```

Comment:
Is there in place for when `internal` or `external` are `null`?

Reply:
Also, is there already an input similar to this somewhere else?

Reply:
IMO if they come across null it should error. Should be an empty array or an array of strings. And no this is slightly different than existing contact inputs